.github/workflows/test_matrix.json

```diff
@@ -13,39 +13,39 @@
     "container": "ghcr.io/agnostiqhq/covalent-dev/debian12-py310:latest",
     "backend": "local",
     "experimental": false,
-    "trigger": ["schedule", "workflow_dispatch", "pull_request"]
+    "trigger": ["schedule", "workflow_dispatch"]
```

Comment:
Would this reduce the test matrix for PRs?

Reply:
Also note that the local and dask tests exercise slightly different codepaths.

Reply:
The local and dask tests do exercise different codepaths. For example, only the local executor uses the endpoints to download and [upload electron assets](https://github.com/AgnostiqHQ/covalent/blob/develop/covalent_dispatcher/_service/assets.py#L82).

Reply:
The local and dask tests do exercise different codepaths. In particular, they test different flows for building and submitting sublattices.

Reply:
That's the objective, I don't want the matrix to be that comprehensive for PRs. I have added the local and Dask again tho.